### PR TITLE
Criteo CTR with Cube Doc

### PR DESCRIPTION
**Pull request by bjjwwang**
1. Cube Local Mode README (Simplified Chinese/ English)
2. Criteo CTR with Cube step by step (Simplified Chinese/ English)
3. some png images necessary

### CODE REVIEW

File: doc/CUBE_LOCAL.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Cube: Sparse Parameter Server (Local Mode)
```
Comment (guru4elephant) [inline doc/CUBE_LOCAL.md]: I think cube is sparse parameter indexing service not parameter server, please refine this concept

File: doc/CUBE_LOCAL_CN.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# 单机版稀疏参数服务器Cube
```
Comment (guru4elephant) [inline doc/CUBE_LOCAL_CN.md]: Cube不是参数服务器

Comment (bjjwwang) [reply doc/CUBE_LOCAL_CN.md]: 谢谢 已更正为 稀疏参数索引服务

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**